R/lazy-join-query.R

```diff
@@ -139,8 +180,12 @@ op_vars.lazy_join_query <- function(op) {
   op$vars$alias
 }
 #' @export
+op_vars.lazy_multi_join_query <- function(op) {
+  op$vars$name
+}
+#' @export
 op_vars.lazy_semi_join_query <- function(op) {
-  names(op$vars)
+  op$vars$name
```

Comment:
I changed the `vars` field of `lazy_semi_join_query` to a tibble with columns `name` and `var`, so that it is more similar to the `vars` field of `lazy_multi_join_query`.